packages/gravity-ui-nunjucks/README.md

```diff
@@ -52,6 +52,16 @@ npm start
 ```
 This should also open the pattern library in your default web browser. In any case, the URL will be listed in the console output. By default it is: http://localhost:3000/
 
+#### Run accessibility tests
+Runs component's accessibility tests using [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs).
+
+First download and install the lastest `chromedriver` from [here](https://chromedriver.storage.googleapis.com/index.html).
+
+[See more on Webdriver requirements and setup](https://selenium.dev/documentation/en/webdriver/driver_requirements/).
+```
+npm run test:a11y
+```
+
```

Comment:
As per my [other comment](https://github.com/buildit/gravity-ui-web/pull/364#issuecomment-555516579), it would be good to add a note here to explain that the pattern library needs to be running before you can launch these tests.

Unless you update the Gulp task to do so automatically... ;-)